File: SEM5-C/ejer4.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEM5_C
{
    internal class ejer4
    {
        static void Main(string[] args)
        {
            Console.Write("Ingrese la cantidad de numeros: ");
            int cant = int.Parse(Console.ReadLine());

            int num, Cceros = 0, Cpares = 0, cImpares = 0;
            Console.WriteLine();
            for (int i = 0; i < cant; i++)
            {
                Console.Write($"Ingrese numero {i + 1}: ");
                num = int.Parse(Console.ReadLine());
                if (num == 0) Cceros++;
                else if (num % 2 == 0) Cpares++;
                else cImpares++;
            }

            Console.WriteLine("\n\nCantidad ceros: " + Cceros);
            Console.WriteLine("\n\nCantidad pares: " + Cpares);
            Console.WriteLine("\n\nCantidad impares: " + cImpares);

        }
    }
}

File: SE5-P/ejer4.py
cant = int(input("Ingrese la cantidad de numeros: "))

Cceros=Cpares=cImpares=0

print ()

for i in range (0, cant):
    num = int(input(f"Ingrese numero {i + 1}: "))
    
    if (num == 0): Cceros+=1
    elif (num % 2 == 0): Cpares+=1
    else: cImpares+=1

print("\n\nCantidad ceros: ", Cceros)
print("Cantidad pares: ", Cpares)
print("Cantidad impares: ", cImpares)